PDFTron/webviewer-ui
# [Feature] Added a way to unlink links

To test:

Scenario 1:
1. create an annotation
2. link it
3. select the annotation created, ensure unlink icon is visible
4. if unlinked, check annotation count
5. make sure original annot is not deleted

Scenario 2:
1. select text
2. link it
3. select the highlight created, ensure unlink icon is visible
4. if unlinked, check annotation count
5. make sure highlight annot is deleted

Scenario 3:
1. create multiple annotations
2. link them
3. select the annotations created, ensure unlink icon is visible
4. if unlinked, check annotation count
5. make sure original annots are not deleted
